api/mocks/FlowAccessClient.go

```diff
@@ -0,0 +1,813 @@
+// Code generated by mockery v2.21.4. DO NOT EDIT.
```

Comment:
was this copied over or generated? if it was copied it's better if generated, just add a comment for generation on top of FlowAccessClient and use mockery.

Reply:
I even think it would be useful to create a PR against flow-go-sdk and generate the mock there for the interface: 
https://github.com/onflow/flow-go-sdk/blob/master/access/client.go#L34

Then you could just use that mock and it would also help others.

Reply:
This was generated with mockery, I have added a relevant Makefile recipe as well in: https://github.com/onflow/flow-evm-gateway/pull/24/commits/1b20bb599f3cfba613f4f65ac61fb6ef00987a4c.
I will also open up a PR to do this in flow-go-sdk, however, it might take some time to get that change in. So for the time being, we can stick to the mock generated in this repository.

Reply:
Ok great. As long as it's being generated here it's all good, and the PR in go-sdk would be good to have but not a blocker yeah